docs/source/conf.py

```diff
@@ -110,7 +110,15 @@
 html_logo = "_static/delta-lake-white.png"
 html_favicon = "_static/favicon.ico"
 
-html_theme_options = {}
+html_theme_options = {
+    "show_toc_level": 2,
+    "use_sidenav_for_toctree": True,
```

Comment:
can comment what each of these setting are for future devs.

Reply:
remove this entire block. its not needed.